content/4_Gold/LIS.mdx

```diff
@@ -426,6 +426,11 @@ class PCB{
 
 <Problems problems="lis" />
 
+<Info>
```

Comment:
Looks good, is there any way to make this a page in the guide? That might be a bit more convenient. Otherwise everything looks great.

Reply:
you read the translation right
and idk how to make this a separate page lol (aren't you a web dev)

Reply:
yeah the translation was good

Reply:
oop, i haven't yet looked at how the general structuring works either

Reply:
maybe we can put this problem on the CF gym

Reply:
if it's on the cf gym we need like test cases